product_docs/docs/edbcloud/beta/overview/03_security_compliance.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: "Security and Compliance"
```

Comment:
We probably shouldn't even talk about compliance here yet.

Reply:
That is the issue of difference between  preview and GA release. We have a similar issue in UI/UX. Many screens in preview release are not up to the mark of basic UX best practices but they will be corrected in GA release. So how we tell users about what we have- do we have a lousy UI screen with list of users which you can't search? Our docs are also kind of marketing materials. I was struggling with that and decided to tell both what is available and what we are planning:. Example is here https://docs.google.com/document/d/1oPvrm5gjxlqYdejFPwvYd42wgcONNsKorZWtNIXQAhI/edit
(Please feel free to comment)

Telling user that our preview release is compliant is wrong. But not telling them that our GA release will be compliant may be also problematic.

Reply:
> Our docs are also kind of marketing materials


The preview customers will all know that compliance is coming - it's front and center for the messaging with them. I don't think we need to worry about that as a marketing point.

Reply:
> However the preview customers will all know that compliance is coming - it's front and center for the messaging with them. I don't think we need to worry about that as a marketing point.

I was thinking about it. The entire section was more a marketing thing written for the right hemisphere of the brain- to make customers to *feel* comfortable with the product. May be we should remove the entire "Security and Compliance"  section form the preview release?

Reply:
I would drop compliance for now, and add it back in once we have a SOC2 report, including how to get a copy etc. We should keep the Security piece, and give customers enough information to inform their own Compliance posture etc.

Reply:
I removed compliance related material.